src/Longitude.cpp

```diff
@@ -2,6 +2,17 @@
 #include "Utility.hpp"
 #include "Longitude.hpp"
 
+// define INDI in the local configuration if you want it to work
+#ifndef INDI
```

Comment:
Could you move this default setting to `Configuration_adv.hpp`? I think the `Misc Configuration` makes sense for this

Reply:
Sure